src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -24,31 +16,62 @@ import LegendEditor from '../../../plots-and-tables/components/LegendEditor';
 import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
 import ColourInversion from '../../../plots-and-tables/components/ColourInversion';
 
-const defaultPlotStylingConfig = {
-  legendEnabled: 'true',
-  legendPosition: 'top',
-  labelsEnabled: true,
-  labelSize: 28,
-  xAxisText: 'Principal Components',
-  yAxisText: 'Proportion of Variance Explained',
-  xDefaultTitle: 'Principal Components',
-  yDefaultTitle: 'Proportion of Variance Explained',
-  titleSize: 12,
-  titleText: 'Scree plot',
-  titleDx: 10,
-  titleAnchor: 'start',
-  masterFont: 'sans-serif',
-  masterSize: 13,
-  xaxisText: 'Principal Components',
-  yaxisText: 'Proportion of Variance Explained',
-  axisTitlesize: 13,
-  axisTicks: 13,
-  axisOffset: 0,
-  transGrid: 10,
-  width: 530,
-  height: 400,
-  maxWidth: 720,
-  maxHeight: 530,
+import isBrowser from '../../../../../../utils/environment';
+
+import loadCellSets from '../../../../../../redux/actions/cellSets/loadCellSets';
+
+import { loadProcessingSettings } from '../../../../../../redux/actions/experimentSettings';
+
+import {
+  loadPlotConfig,
+} from '../../../../../../redux/actions/componentConfig/index';
+
+import CalculationConfig from './CalculationConfig';
+import fakeData from './fake_new_data.json';
```

Comment:
Are we using fake data for now?

Reply:
It was being used from before too, we need to use fake data for the elbow plot because it displays information about a the result of pca (which is not implemented in the worker yet)

Reply:
Right, thanks for the explanation!